### PR TITLE
Fix typos.  Small proposed re-wording.

### PR DESCRIPTION
**Pull request by DanAtShenTech**

### CODE REVIEW

File: docs/source/menupage.rst [hunk 1]
```diff
@@ -195,7 +195,7 @@ Wagtail has a restriction that forbids models from subclassing more than one oth
 
 .. _manipulating_submenu_items:
 
-Using ``MenuPage`` to manipulating sub-menu items
+Using ``MenuPage`` to manipulate sub-menu items
 =================================================
```
Comment (ababic) [inline docs/source/menupage.rst]: @DanAtShenTech Tiny thing, but can we knock a couple of underline characters from here to match the heading length?

**Comment by ababic** [comment]
Thanks for this @DanAtShenTech. I really appreciate your help in making the language in these docs flow better. Language/writing is definitely not a strong point of mine.

File: docs/source/menupage.rst (hunk 1 above)
Comment (DanAtShenTech) [reply docs/source/menupage.rst]: Fixed now @ababic

**Comment by ababic** [comment]
Merged in b9fd6ba. Thanks again @DanAtShenTech :)

**Comment by DanAtShenTech** [comment]
I think the docs are great @ababic.  There's always a few typos to be fixed.  I'm glad to be able to contribute.

**Pull request merged**